docs/src/examples/expect/example.md

```diff
@@ -0,0 +1,37 @@
+## Examples
+### Use different matchers
+
+This example shows how to use the `expect` function and its different matchers
```

Comment:
I would remove the "This example shows blah blah" From ALL examples, feels repetitive and doesn't really add information.

This one, for example could become:

Use the `expect` function and its different matchers to assert conditions.